**Zum Spielen**

In [14]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

Als erstes importieren wir unser github Repo. Der nächste Schritt ist der import des Datasets.


In [11]:
#!git clone https://github.com/DeamonAmelie/heart
#!git pull https://github.com/DeamonAmelie/heart
!wget https://raw.githubusercontent.com/DeamonAmelie/heart/main/processed.cleveland.data.csv
!pwd
!ls

--2021-05-19 15:44:30--  https://raw.githubusercontent.com/DeamonAmelie/heart/main/processed.cleveland.data.csv
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 18461 (18K) [text/plain]
Saving to: ‘processed.cleveland.data.csv.1’

processed.cleveland 100%[===================>]  18.03K  --.-KB/s    in 0.001s  

2021-05-19 15:44:31 (26.5 MB/s) - ‘processed.cleveland.data.csv.1’ saved [18461/18461]

/content
heart			    processed.cleveland.data.csv
processed.cleveland.data    processed.cleveland.data.csv.1
processed.cleveland.data.1  sample_data


In [15]:
#df = pd.read_csv("heart/heart.csv")
df = pd.read_csv("processed.cleveland.data.csv")
df.head()

63.0  1.0  1.0.1  145.0  233.0  1.0.2  ...  0.0  2.3  3.0  0.0.1  6.0  0
0  67.0  1.0    4.0  160.0  286.0    0.0  ...  1.0  1.5  2.0    3.0  3.0  2
1  67.0  1.0    4.0  120.0  229.0    0.0  ...  1.0  2.6  2.0    2.0  7.0  1
2  37.0  1.0    3.0  130.0  250.0    0.0  ...  0.0  3.5  3.0    0.0  3.0  0
3  41.0  0.0    2.0  130.0  204.0    0.0  ...  0.0  1.4  1.0    0.0  3.0  0
4  56.0  1.0    2.0  120.0  236.0    0.0  ...  0.0  0.8  1.0    0.0  3.0  0

[5 rows x 14 columns]

## Beschreibung der Daten

age - Age of the patient

sex - Sex of the patient

cp - Chest pain type ~ 0 = Typical Angina, 1 = Atypical Angina, 2 = Non-anginal Pain, 3 = Asymptomatic

trtbps - Resting blood pressure (in mm Hg)

chol - Cholestoral in mg/dl fetched via BMI sensor

fbs - (fasting blood sugar > 120 mg/dl) ~ 1 = True, 0 = False

restecg - Resting electrocardiographic results ~ 0 = Normal, 1 = ST-T wave normality, 2 = Left ventricular hypertrophy

thalachh - Maximum heart rate achieved

oldpeak - Previous peak

slp - Slope

caa - Number of major vessels

thall - Thalium Stress Test result ~ (0,3)

exng - Exercise induced angina ~ 1 = Yes, 0 = No

output - Target variable, 1=High chance of heart attack, 0=low chance heart attack



## EDA 


*   Daten Darstellen
*   Daten transfrmieren ?
* Daten nochmal darstellen
* korrelation darstellen

